README.md

```diff
@@ -35,9 +35,17 @@ make run
 
 2. When connections are established, you can access:
 
-    * the LMS server on [localhost:8072](http://localhost:8072)
+    * the LMS server on [localhost:8072/login](http://localhost:8072/login)
```

Comment:
I'm not able to access `localhost:8072`, it redirects to `richi` which is not available in my case. Did I missed something here ?

Reply:
It's a point I have to fix, I changed the image version of openedx and the current one comprises the redirection to `richie`